File: bespin/src/test/java/org/example/product/bespin/test/BespinModuleTest.java
package org.example.product.bespin.test;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.example.product.corellia.fixtures.CorelliaModuleFixture;
import org.junit.jupiter.api.Test;

public class BespinModuleTest {

    @Test
    void testModule() {
        assertEquals(3, new CorelliaModuleFixture().threeModules().size());
    }
}

File: bespin/src/test/kotlin/org/example/product/bespin/test/BespinModuleTest.kt
package org.example.product.bespin.test

import org.example.product.corellia.fixtures.CorelliaModuleFixture
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class BespinModuleTest {
    @Test
    fun testModule() {
        Assertions.assertEquals(3, CorelliaModuleFixture().threeModules().size)
    }
}
